Common/Product/SharedProject/CommonPropertyPage.cs

```diff
@@ -271,12 +271,12 @@ public abstract class CommonPropertyPage : IPropertyPage, IDisposable {
         }
 
         int IPropertyPage.Apply() {
-            try {
+            //try {
                 Apply();
                 return VSConstants.S_OK;
-            } catch (Exception e) {
-                return Marshal.GetHRForException(e);
-            }
+            //} catch (Exception e) {
+            //    return Marshal.GetHRForException(e);  // catching exception causes VS to ignore it completely, throw is important!
+            //}
```

Comment:
Just go through and delete code rather than commenting it out. If it deserves a comment (like this one does), put it above the code to mention that we're letting exceptions fall through on purpose.